src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -205,13 +199,13 @@ def body(self):
         self.cmd('network express-route circuit list --resource-group {}'.format(self.resource_group),
             checks=NoneCheck())
 
-class NetworkExpressRouteCircuitScenarioTest(VCRTestBase):
+class NetworkExpressRouteCircuitScenarioTest(ResourceGroupVCRTestBase):
 
     def __init__(self, test_method):
          # The resources for this test did not exist so the commands will return 404 errors.
          # So this test is for the command execution itself.
```

Comment:
@BurtBiel is this still true? There's an express-route circuit create. Looks like maybe we should create a separate issue to revise this test.